Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
@dsjen I thought about edx-lint initially, but that seems very PyLint oriented at the moment. If we want a common file then having it in the UI Toolkit seems best because that does get installed. However, as you point out, it may be that every repo wants its own subtle variations, in which case having it in the cookiecutter repo makes more sense.

Note: I have a T&L PR to add front end capabilities to the cookiecutter repo:

https://openedx.atlassian.net/browse/TNL-3681